.github/workflows/ci.yml

```
@@ -0,0 +1,324 @@
+name: CI/CD Pipeline
+
+on:
+  push:
+    branches: [ main, develop, 'feature/**', 'hotfix/**' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)